sambaplayersdk/src/main/java/com/sambatech/player/SambaPlayer.java

```diff
@@ -440,8 +443,8 @@ else if (jsonObject.has("type")) {
 		@Override
 		public void onDisconnected() {
 			player.setControlsVisible(true,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.qa.sambatech.com/sonar/coding_rules#rule_key=squid%3AS1067)